src/webapp/src/components/Navbar/index.tsx

```diff
@@ -1,23 +1,39 @@
 import React from 'react';
 import { Button } from '@small-ads/ui';
 import { useHistory, useRouteMatch } from 'react-router-dom';
-import { HeaderStyles, Header } from './navbar.style';
+import { useDispatch, useSelector } from 'react-redux';
+import { signOutUser } from '../../auth/asyncAuthActions';
+import { Header, HeaderStyles } from './navbar.style';
+import { selectUser } from '../../auth/selectors';
 
 export const Navbar = () => {
   const history = useHistory();
   const route = useRouteMatch('/login');
+  const dispatch = useDispatch();
+  const { authStatus } = useSelector(selectUser);
+
+  const handleLogout = () => {
+    dispatch(signOutUser());
+  };
   return (
     <HeaderStyles>
       <Header>
-        <Button
-          disabled={Boolean(route?.isExact)}
-          size='sm'
-          onClick={() => {
-            history.push('/login');
-          }}
-        >
-          Login
-        </Button>
+        {!authStatus && (
+          <Button
+            disabled={Boolean(route?.isExact)}
+            size='sm'
+            onClick={() => {
+              history.push('/login');
+            }}
```

Comment:
We have two onClick handlers in this file. One has an inline function and one is not. Perhaps we can decide on a convention and follow it accross the project. What you think?

Reply:
Of course, it makes only sense to be consistent. I am going to go with **not** using inline functions because if the functions are too long it can obscure the code.